hw/ip/hmac/dv/cov/hmac_cov_excl.el

```diff
@@ -2,37 +2,82 @@
 Block 30 "0" "assign gen_normal_fifo.storage_rdata = gen_normal_fifo.storage[0];"
+ANNOTATION: "[UNR] u_rspfifo is not used in tlul_adapter"
 Block 26 "3968208487" "gen_normal_fifo.fifo_rptr <= (gen_normal_fifo.fifo_rptr + {{(gen_normal_fifo.PTR_WIDTH - 1) {1'b0}}, 1'b1});"
+ANNOTATION: "[UNR] u_rspfifo is not used in tlul_adapter"
 Block 25 "3085096792" "gen_normal_fifo.fifo_rptr <= {(~gen_normal_fifo.fifo_rptr[(gen_normal_fifo.PTR_WIDTH - 1)]), {(gen_normal_fifo.PTR_WIDTH - 1) {1'b0}}};"
+ANNOTATION: "[UNR] u_rspfifo is not used in tlul_adapter"
 Block 24 "2927261095" "if ((gen_normal_fifo.fifo_rptr[0] == (Depth - 1)))"
+ANNOTATION: "[UNR] u_rspfifo is not used in tlul_adapter"
 Block 17 "3894529251" "gen_normal_fifo.fifo_wptr <= (gen_normal_fifo.fifo_wptr + {{(gen_normal_fifo.PTR_WIDTH - 1) {1'b0}}, 1'b1});"
+ANNOTATION: "[UNR] u_rspfifo is not used in tlul_adapter"
 Block 16 "1403547534" "gen_normal_fifo.fifo_wptr <= {(~gen_normal_fifo.fifo_wptr[(gen_normal_fifo.PTR_WIDTH - 1)]), {(gen_normal_fifo.PTR_WIDTH - 1) {1'b0}}};"
 CHECKSUM: "3885231045 1022284358"
 INSTANCE: tb.dut.u_reg.u_socket.err_resp
 ANNOTATION: "[UNR] Error response is captured in tlul_interface level, no error will land here in hmac"
 Block 4 "3222429784" "err_req_pending <= 1'b1;"
+ANNOTATION: "[UNR] error resp is handled at tlul interface level"
 Block 16 "2498388664" "err_rsp_pending <= 1'b1;"
+CHECKSUM: "1352181506 3803838024"
+INSTANCE: tb.dut.u_reg.u_wipe_secret
+ANNOTATION: "[UNR] qe is a constant, the coverage tool did not recognize it"
+Block 3 "0" "assign qe = we;"
+CHECKSUM: "1352181506 3803838024"
+INSTANCE: tb.dut.u_reg.u_key0
+ANNOTATION: "[UNR] qs is a constant, the coverage tool did not recognize it"
+Block 1 "0" "assign qs = d;"
+CHECKSUM: "1352181506 3803838024"
+INSTANCE: tb.dut.u_reg.u_key1
+ANNOTATION: "[UNR] qs is a constant, the coverage tool did not recognize it"
+Block 1 "0" "assign qs = d;"
+CHECKSUM: "1352181506 3803838024"
+INSTANCE: tb.dut.u_reg.u_key2
+ANNOTATION: "[UNR] qs is a constant, the coverage tool did not recognize it"
+Block 1 "0" "assign qs = d;"
+CHECKSUM: "1352181506 3803838024"
+INSTANCE: tb.dut.u_reg.u_key3
+ANNOTATION: "[UNR] qs is a constant, the coverage tool did not recognize it"
+Block 1 "0" "assign qs = d;"
+CHECKSUM: "1352181506 3803838024"
+INSTANCE: tb.dut.u_reg.u_key4
+ANNOTATION: "[UNR] qs is a constant, the coverage tool did not recognize it"
+Block 1 "0" "assign qs = d;"
+CHECKSUM: "1352181506 3803838024"
+INSTANCE: tb.dut.u_reg.u_key5
+ANNOTATION: "[UNR] qs is a constant, the coverage tool did not recognize it"
+Block 1 "0" "assign qs = d;"
+CHECKSUM: "1352181506 3803838024"
+INSTANCE: tb.dut.u_reg.u_key6
+ANNOTATION: "[UNR] qs is a constant, the coverage tool did not recognize it"
+Block 1 "0" "assign qs = d;"
+CHECKSUM: "1352181506 3803838024"
+INSTANCE: tb.dut.u_reg.u_key7
+ANNOTATION: "[UNR] qs is a constant, the coverage tool did not recognize it"
+Block 1 "0" "assign qs = d;"
```

Comment:
Hi @eunchan , these are new exclusions that we have not reviewed. They are due to the coverage tool limitations, when you assign a constant, the line coverage will show uncovered. 
For example for `key0` in `prim_subreg_ext.sv`:
```
  assign qs = d;  // this is shown as a coverage hole because qs is never changed.
  assign q = wd;
  assign qe = we;
  assign qre = re;
```
I waived them. Please let me know if you have any concerns.